bin/rails

```diff
@@ -4,6 +4,7 @@
 
 ENGINE_ROOT = File.expand_path('../..', __FILE__)
 ENGINE_PATH = File.expand_path('../../lib/scim_rails/engine', __FILE__)
+APP_PATH = File.expand_path('../../spec/dummy/config/application', __FILE__)
```

Comment:
This tells rails where the dummy application is located. You wouldn't need this if we were using the default "test" directory but we do because we are using "spec" instead.

Interestingly, you can supply a flag when generating the engine to customize the location of the dummy application if you want it to be somewhere other than test. I learn that reading the rails source code today!